webapp/javascript/redux/reducers/continuous.ts

```diff
@@ -845,8 +841,14 @@ export const continuousSlice = createSlice({
     /*      Timeline Sides       */
     /*****************************/
     builder.addCase(fetchSideTimelines.pending, (state) => {
-      state.leftTimeline = { ...state.leftTimeline, type: 'loading' };
-      state.rightTimeline = { ...state.rightTimeline, type: 'loading' };
+      state.leftTimeline = {
+        ...state.leftTimeline,
+        type: getNextStateFromPending(state.leftTimeline.type),
+      };
+      state.rightTimeline = {
+        ...state.rightTimeline,
+        type: getNextStateFromPending(state.leftTimeline.type),
```

Comment:
does `type` mean loading type? does it make sense to name this prop something like `loadingType`?

what the point to track next state from pending?

Reply:
`type` means 
<img width="635" alt="Screen Shot 2022-10-13 at 13 40 25" src="https://user-images.githubusercontent.com/6951209/195655420-8c1028aa-4ef3-48cf-90b9-e38dde7b64c1.png">

> what the point to track next state from pending?
It's just an utility. Imagine a state machine where given current state + action is `pending`, it returns what the next state should be. Its logic basically is: do we already have data? If so we are `reloading`, otherwise we are just `loading`.